Makefile

```diff
@@ -435,7 +435,11 @@ e2e-framework: ## Builds the CAPI e2e framework
 
 ARTIFACTS ?= ${ROOT_DIR}/_artifacts
 
-KUBEBUILDER_ASSETS ?= $(shell $(SETUP_ENVTEST) use --use-env -p path $(KUBEBUILDER_ENVTEST_KUBERNETES_VERSION))
+ifeq ($(shell go env GOOS),darwin) # Use the darwin/amd64 binary until an arm64 version is available
+	KUBEBUILDER_ASSETS ?= $(shell $(SETUP_ENVTEST) use --use-env -p path --arch amd64 $(KUBEBUILDER_ENVTEST_KUBERNETES_VERSION))
```

Comment:
Just out of curiosity. Mac will just work with the amd64 binaries on arm because they automatically do some emulation magic?

Reply:
Yes, amd64 binaries work seamlessly on Apple Silicon thanks to [Rosetta 2](https://support.apple.com/en-us/HT211861).